go-app-ussd_nurse.js

```diff
@@ -3,22 +3,114 @@ go;
 
 go.app = function() {
     var vumigo = require("vumigo_v02");
+    var _ = require('lodash');
     var App = vumigo.App;
-    var EndState = vumigo.states.EndState;
+    var Choice = vumigo.states.Choice;
+    var ChoiceState = vumigo.states.ChoiceState;
+    var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
+    // var EndState = vumigo.states.EndState;
+    var JsonApi = vumigo.http.api.JsonApi;
+
+    var SeedJsboxUtils = require('seed-jsbox-utils');
+    var IdentityStore = SeedJsboxUtils.IdentityStore;
+
+    var utils = SeedJsboxUtils.utils;
 
     var GoNDOH = App.extend(function(self) {
-        App.call(self, "state_start");
+        App.call(self, "state_route");
         var $ = self.$;
 
+        // variables for services
+        var is;
+
         self.init = function() {
+            // initialising services
+            is = new IdentityStore(
+                new JsonApi(self.im, {}),
+                self.im.config.services.identity_store.token,
+                self.im.config.services.identity_store.url
+            );
+
+            var msisdn = utils.normalize_msisdn(self.im.user.addr, '27');
+            return is
```

Comment:
I don't think we should put this stuff (starting with the `return`) in `init` - I think `init` will run everytime you switch states, so it'll make multiple api calls.